### PR TITLE
Add mentorship code of conduct

### PR DESCRIPTION
**Pull request by goelsonali**
## Description
 
Code of conduct for mentee and mentor

#32

## Change Type
- [ ] Bug Fix
- [x] New Feature
- [ ] Code Refactor
- [ ] Mentor Update
- [ ] Documentation
- [ ] Other

## Screenshots

*before*
Screenshot 2024-04-24 at 16 12 20

*after*
Screenshot 2024-04-24 at 16 10 23
Screenshot 2024-04-24 at 16 10 33

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] I checked and followed the [contributor guide](CONTRIBUTING.md) 
- [x] I have tested my changes locally.
- [x] I have added a screenshot from the website after I tested it locally

### CODE REVIEW

File: mentorship-code-of-conduct.html [hunk 1]
```diff
@@ -6,6 +6,56 @@
...
+            <li>Cancel or reschedule appointments with a minimum of 24 hours prior notice by providing a reason using preferable channel (email, slack, or anything else you both chose as the channel of communication) to the mentor.</li>
+            <li>No-shows are unacceptable for mentors and mentees. No-shows should be informed to the Mentorship Programme Team by <a href="https://bit.ly/women_coding_community">Slack </a>, channel #mentorship. The Mentorship Programme Team will evaluate case by case, and this is one of the possible actions that could result in exclusion from the current and future mentorship program cycles.</li>
+            <li>No-replies by email or slack to the Mentorship Programme Team and/or mentor in a week are unacceptable during the program. Participants who applied but failed to reply without any reason provided will be banned from the current cycle.</li>
+            <li>Create a <a href="https://bit.ly/women_coding_community">Slack Account</a>, join the #mentorship channel and check slack messages in the program to remain updated with all communication.</li>
```
Comment (lilyndk) [inline mentorship-code-of-conduct.html]: add 'target="_blank" for external links to have those links open in a new window

File: mentorship-code-of-conduct.html [hunk 2]
```diff
@@ -6,6 +6,56 @@
...
+                    <li>It is <b>mandatory</b> to provide feedback using google forms shared with you before the session.</li>
+                    <li>If you can't attend the session, explain your reason in reply to the invitation.</li>
+                </ul>
+            </li>
+            <li>Discussions between you and your mentee <b>are considered to be confidential</b>. Be careful about sensitive personal issues. Do not share any specific content of the sessions or personal data without the permission of your mentee. </li>
+            <li>Please ensure that your profile on the mentorship page is kept up-to-date.</li>
+        </ol>
+    </div>
+    
+    <div class="col-12 section global-code">
```
Comment (lilyndk) [inline mentorship-code-of-conduct.html]: This 'global-code' class seems unnecessary. I suggest removing it, if it's not related to any styles or functionality :)

File: mentorship-code-of-conduct.html (hunk 1 above)
Comment (dricazenck) [inline mentorship-code-of-conduct.html]:
```suggestion
            <li>Create a <a target="blank" href="https://bit.ly/women_coding_community">Slack Account</a>, join the #mentorship channel and check slack messages in the program to remain updated with all communication.</li>
```

File: mentorship-code-of-conduct.html (hunk 2 above)
Comment (dricazenck) [reply mentorship-code-of-conduct.html]:
```suggestion
    <div class="col-12 section">
```

**Review by dricazenck — APPROVED** [review]

**Pull request merged**